modules/custom/az_paragraphs/az_paragraphs_splitscreen/css/az_splitscreen.css

```diff
@@ -0,0 +1,37 @@
+/* Mobile/Base styles */
```

Comment:
Nitpick: Looks like this file is using tabs instead of spaces.  We should change to spaces before merging.